lib/mini_profiler/config.rb

```diff
@@ -86,6 +86,24 @@ def self.default
 
       attr_reader :assets_url
 
+      def explicit_authorization_mode?
+        authorization_mode == :allow_authorized
+      end
```

Comment:
lets remove this method, then we are good to merge!

Reply:
Can't believe I missed that -- thank you!